backend/danswer/dynamic_configs/store.py

```diff
@@ -82,19 +46,36 @@ def store(self, key: str, val: JSON_ro, encrypt: bool = False) -> None:
             session.commit()
 
     def load(self, key: str) -> JSON_ro:
+        redis_value = self.redis_client.get(key)
+        if redis_value:
+            if isinstance(redis_value, bytes):
+                return json.loads(redis_value.decode("utf-8"))
+            elif isinstance(redis_value, str):
```

Comment:
Doesn't redis_client.get always return type bytes?

Reply:
Ya, looked closer, seems good, I dont think it should ever be str